slasher/detection/attestations/spanner.go

```diff
@@ -219,13 +234,18 @@ func (s *SpanDetector) updateMinSpan(ctx context.Context, att *ethpb.IndexedAtte
 				indices = append(indices, idx)
 			}
 		}
-		if err := s.slasherDB.SaveEpochSpansMap(ctx, epoch, spanMap); err != nil {
-			return err
-		}
-		if len(indices) == 0 {
-			break
+		//copy(valIndices, indices)
```

Comment:
```suggestion
```

Reply:
it should improve performance. commented it out for some reason